core/src/main/scala/org/apache/spark/util/io/ChunkedByteBuffer.scala

```diff
@@ -40,6 +41,9 @@ private[spark] class ChunkedByteBuffer(var chunks: Array[ByteBuffer]) {
   require(chunks != null, "chunks must not be null")
   require(chunks.forall(_.position() == 0), "chunks' positions must be 0")
 
+  // Chunk size in bytes
+  private val NIO_BUFFER_LIMIT = SparkEnv.get.conf.get(config.STORAGE_NIO_BUFFER_LIMIT)
```

Comment:
`SparkEnv.get` may be null in test cases, how about `Option(SparkEnv.get).map(_.conf.get(BUFFER_WRITE_CHUNK_SIZE)).getOrElse(BUFFER_WRITE_CHUNK_SIZE.defaultValue.get)`